config/channel/consolidated/deployments/dispatcher.yaml

```diff
@@ -29,6 +29,8 @@ spec:
       messaging.knative.dev/role: dispatcher
   template:
     metadata:
+      annotations:
+        kafka.eventing.knative.dev/configmap-hash: ''
```

Comment:
What do you think of adding a comment about what's this used for and to not change it manually?

Reply:
+1

Reply:
Added the comment now